app/database_etl/postgres_tables_handler/table_generator.py

```diff
@@ -0,0 +1,139 @@
+import logging
+from uuid import uuid4
+from datetime import datetime
+
+from app.utils.notifications_sender import send_slack_message
+from app.utils import airtable_fields_config
+from app.database_etl.location_utils import add_latlng_to_df, get_country_code
+from .table_formatter import replace_null_string
+
+import pandas as pd
+
+
+def isotype_col(isotype_string, x):
+    # Function for creating isotype boolean columns
+    if x:
+        return True if isotype_string in x else False
+    return False
+
+
+def create_dashboard_source_df(original_data, current_time):
```

Comment:
hmm consider making this into a class so that we can reference self.current_time instead of having to add it as an arg to every function call

Reply:
making the dashboard source as a class or making a whole class for etl?